client/src/pages/City.jsx

```diff
@@ -1,9 +1,21 @@
-import React from 'react';
-import ScrollSpy from '../components/common/ScrollSpy/ScrollSpy';
+import React, { useContext } from 'react';
+import StoryCard from '../components/common/Story';
+import { AuthContext } from '../context/AuthContext';
+import { CityContext } from '../context/CityContext';
 
 function City() {
+  const { userData } = useContext(AuthContext);
+  const { cityData: cityInfo } = useContext(CityContext);
+  const { cityData, stories } = cityInfo;
+
   return (
-    <ScrollSpy />
+    <div>
+      {stories
+    && stories.length
```

Comment:
```suggestion
      {stories?.length
```
just as a suggestion 
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining
check this link might be useful for you

Reply:
done, Thank you